File: pascal.py
import sys

n = int(sys.argv[1])
for i in range(1,n+1):
	c=1
	s=''
	for j in range(n-i-1):
		s+=" "
	print s
	s=''
	for k in range(1,n+1):
		s+=str(c)+" "
		c=c*(i-k)/k
	print s
	print "\n"

File: pascal.php
<?php
$n = $argv[1];

for($i=1;$i<=$n;$i++)
{	
	$c=1;
	for($j=1;$j<=($n-$i);$j++)
	{
		echo " ";
	}	
	for($k=1;$k<=$i;$k++)
	{

	echo $c." ";
	$c = $c*($i-$k)/$k;
	}
	echo "\n";
}
?>